kondo_driver:
  serial_port: /dev/ttyUSB0
  baudrate: 15000
  servo_drivers:
    head_yaw_driver:             {id: 1, dir: 1, offset: 0}
    left_shoulder_pitch_driver:  {id: 2, dir: 1, offset: 0}
    left_shoulder_roll_driver:   {id: 3, dir: 1, offset: 0}
    left_elbow_pitch_driver:     {id: 4, dir: 1, offset: 0}
    left_waist_yaw_driver:       {id: 5, dir: 1, offset: 0}
    left_waist_roll_driver:      {id: 6, dir: 1, offset: 0}
    left_waist_pitch_driver:     {id: 7, dir: 1, offset: 0}
    left_knee_pitch_driver:      {id: 8, dir: 1, offset: 0}
    left_ankle_pitch_driver:     {id: 9, dir: 1, offset: 0}
    left_ankle_roll_driver:      {id: 10, dir: 1, offset: 0}
    right_shoulder_pitch_driver: {id: 11, dir: 1, offset: 0}
    right_shoulder_roll_driver:  {id: 12, dir: 1, offset: 0}
    right_elbow_pitch_driver:    {id: 13, dir: 1, offset: 0}
    right_waist_yaw_driver:      {id: 14, dir: 1, offset: 0}
    right_waist_roll_driver:     {id: 15, dir: 1, offset: 0}
    right_waist_pitch_driver:    {id: 16, dir: 1, offset: 0}
    right_knee_pitch_driver:     {id: 17, dir: 1, offset: 0}
    right_ankle_pitch_driver:    {id: 18, dir: 1, offset: 0}
    right_ankle_roll_driver:     {id: 19, dir: 1, offset: 0}
